### PR TITLE
[FIX] serverStatus: update fee on net connected/disconnected (RT-3323)

### PR DESCRIPTION
**Pull request by rht**
This should be independent of auth connection status

### CODE REVIEW

**Comment by mrajvanshy** [comment]
You need to also watch on fee otherwise the popup will never show the fee because the client is connected before the fee is calculated

**Comment by rht** [comment]
Fixed the other problem (because ng-messages only checks from forms).

File: src/js/controllers/navbar.controller.js [hunk 1]
```diff
@@ -32,27 +32,28 @@ module.controller('NavbarCtrl', ['$scope', '$element', '$compile', 'rpId',
...
-    } 
-    else if ($scope.currentFee && $scope.connected && ($scope.currentFee >= ripple.Amount.from_json(Options.max_tx_network_fee).to_human())) {
-      $scope.serverLoad = 'highLoad';
-      $scope.serverStatus = 'highLoad';
-    }
-    else if ($scope.currentFee && $scope.connected) {
-      $scope.serverLoad = '';
-      $scope.serverStatus = 'lowLoad';
+    else if ($scope.connected && $scope.fee) {
+      if (($scope.fee > Options.low_load_threshold) && ($scope.fee < Options.max_tx_network_fee)) {
```
Comment (mrajvanshy) [inline src/js/controllers/navbar.controller.js]: Options.low_load_threshold is 0.012 so in this if statement you have to make $scope.fee (ripple.Amount.from_json($scope.fee).to_human()) to be able to compare tot he 0.012

File: src/jade/client/navbar.jade [hunk 2]
```diff
@@ -41,20 +41,21 @@ nav.navbar(role="navigation", ng-controller="NavbarCtrl")
...
-              span(ng-if="serverStatus == 'lowLoad'"
-                  rp-popover, rp-popover-placement="bottom", rp-popover-trigger="hover", rp-popover-delay
-                  l10n-rp-popover-content="Connected to the Ripple network. Fee: {{ currentFee }} XRP")
-              span(ng-if="serverStatus == 'connected'"
+              span(ng-switch-when='highLoad'
+                  rp-popover, rp-popover-placement='bottom', rp-popover-trigger='hover', rp-popover-delay
+                  l10n-rp-popover-content="Network fees are currently higher than your maximum. Fee: {{ fee | rpamount }} XRP. <a href='#/advanced'>Change fees</a>")
+              span(ng-switch-when='mediumLoad'
+                  rp-popover, rp-popover-placement='bottom', rp-popover-trigger='hover', rp-popover-delay
+                  l10n-rp-popover-content="'Network fees are currently higher than normal. Fee: {{ fee | rpamount }} XRP'")
```
Comment (mrajvanshy) [inline src/jade/client/navbar.jade]: Please take out the double quotes in the message on this line

**Comment by rht** [comment]
Currently it should work.

Possible explanation:
The fee calculation is instantaneous once connected because ```_computeFee()``` uses

```
Transaction.prototype._getFeeUnits =
Transaction.prototype.feeUnits = function() {
  return Transaction.fee_units['default'];
};
```

```fee_units['default']``` has its value populated on connected.

**Comment by mrajvanshy** [comment]
Yeah I think it works now except could you address the 2 comments above?

File: src/jade/client/navbar.jade (hunk 2 above)
Comment (rht) [inline src/jade/client/navbar.jade]: When I did that the jade engine threw syntax error.

File: src/js/controllers/navbar.controller.js (hunk 1 above)
Comment (rht) [inline src/js/controllers/navbar.controller.js]: Sorry I missed this one. ```max_tx_network_fee``` is set in drops.

File: src/jade/client/navbar.jade (hunk 2 above)
Comment (mrajvanshy) [inline src/jade/client/navbar.jade]: If you take the single quote out and the beginning and end of the message, it shouldn't throw an error. I just tested it

**Comment by rht** [comment]
Ok I removed the single quote. l10n-jade throws syntax error (in somewhere else in the file) if the double quotes are changed to single quote.

**Comment by mrajvanshy** [comment]
LGTM

**Comment by vhpoet** [comment]
LGTM

**Pull request merged**